xmrstak/backend/nvidia/autoAdjust.hpp

```diff
@@ -60,12 +60,10 @@ class autoAdjust
             ctx.device_bfactor = 6;
             ctx.device_bsleep = 25;
 #endif
-            if( cuda_get_deviceinfo(&ctx) != 1 )
-            {
-                printer::inst()->print_msg(L0, "Setup failed for GPU %d. Exitting.\n", i);
-                std::exit(0);
-            }
-            nvidCtxVec.push_back(ctx);
+            if(cuda_get_deviceinfo(&ctx) == 0)
```

Comment:
Can you fix the space indents here, then cherrry-pick 96fa0cec onto a fresh branch (fixing this will cause merge conflict with #116).